flow/Histogram.h

```diff
@@ -93,7 +98,7 @@ class Histogram final : public ReferenceCounted<Histogram> {
 		HistogramRegistry& registry = GetHistogramRegistry();
 		Histogram* h = registry.lookupHistogram(name);
 		if (!h) {
-			h = new Histogram(group_str, op_str, unit, registry, lower, upper);
+			h = new Histogram(Reference<HistogramRegistry>::addRef(&registry), group_str, op_str, unit, lower, upper);
```

Comment:
I'm actually a little confused on why this compiles, because `&registry` is a `Reference<HistogramRegistery> *` which doesn't seem to be convertible to a `HistogramRegistry *` which is what the static `addRef` requires.

In any case, this should be able to be simplified to just

```suggestion
			h = new Histogram(registry, group_str, op_str, unit, lower, upper);
```

Because the `registry` member is already the type that the constructor needs.